tools/dsal/scripts/open-ssh.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -x
+
+sgs=`openstack security group list -f value | grep OpenShift | cut -f 1 -d " "`
+
+for sg in $sgs; do
+    openstack security group rule create --dst-port 22 --protocol tcp $sg
```

Comment:
dev-install already creates a security group for that:
https://github.com/shiftstack/dev-install/blob/eda1ce9b57866defbd76868984a50d06974d1eec/playbooks/prepare_stack.yaml#L126

Reply:
Doesn't seem like it's opening traffic to OpenShift VMs? I still needed to run my script once my OCP cluster was up to be able to access master and worker VMs.